modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,102 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+If {productname} core compontents are upgraded to new major versions, {productname} Server upgrades are considered as major.
+This is the case, if a version upgrade of PostgreSQL, Salt, or openSUSE Leap is needed.
+openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see the Startup Guide at https://doc.opensuse.org.
+
+[IMPORTANT]
+====
+For more information, see the release notes at https://www.uyuni-project.org/pages/stable-version.html.
+Reading the release notes is critical.
+Sometimes additional manual steps are required.
+====
+
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
+
+[NOTE]
+====
+Before running the [command]``server-migrator.sh`` script, check whether the most recent version of the [package]``susemanager`` package is installed:
++
+----
+zypper ref
+zypper up susemanager
+----
+====
+
+The [command]``server-migrator.sh`` script will migrate {productname} Server to the latest version.
+This implies upgrading the underlying operating system to version 15.2.
+
+[WARNING]
+====
+During migration the spacewalk services need to be stopped.
+The script will do this.
+After successful migration the {productname} Server needs to be rebooted manually.
+====
+
+[IMPORTANT]
+====
+Because you cannot fix any issues during migration, make sure you have a backup before starting the migration.
+If you are running {productname} Server on a virtual machine, it is recommended to create a snapshot before performing the migration.
+====
```

Comment:
is there any way we can put all these admons above the migration procedure, and maybe take them out of admons except for the most important ones?